<?xml version="1.0" encoding="UTF-8"?><testsuite xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/jenkinsci/xunit-plugin/ae25da5089d4f94ac6c4669bf736e4d416cc4665/src/main/resources/org/jenkinsci/plugins/xunit/types/model/xsd/junit-10.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="omni" tests="54" failures="7" errors="0" time="66.909" skipped="3" timestamp="2023-11-15T11:03:52"><testcase name="testScreencapture" time="0.742" classname="com.example.ScreenshotTest"><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.io.IOException: The directory /storage/emulated/0/Pictures/screenshots does not exist and could not be created or is not writable.
	at androidx.test.runner.screenshot.BasicScreenCaptureProcessor.process(BasicScreenCaptureProcessor.java:72)
	at com.malinskiy.adam.junit4.android.screencapture.AdamScreenCaptureProcessor.process(AdamScreenCaptureProcessor.kt:47)
	at androidx.test.runner.screenshot.ScreenCapture.process(ScreenCapture.java:144)
	at androidx.test.runner.screenshot.ScreenCapture.process(ScreenCapture.java:126)
	at com.example.ScreenshotTest$testScreencapture$1$1.invoke(ScreenshotTest.kt:32)
	at com.example.ScreenshotTest$testScreencapture$1$1.invoke(ScreenshotTest.kt:31)
	at io.github.kakaocup.kakao.common.views.KDSLView.invoke(KDSLView.kt:12)
	at com.example.ScreenshotTest$testScreencapture$1.invoke(ScreenshotTest.kt:31)
	at com.example.ScreenshotTest$testScreencapture$1.invoke(ScreenshotTest.kt:30)
	at io.github.kakaocup.kakao.screen.Screen.invoke(Screen.kt:119)
	at com.example.ScreenshotTest.testScreencapture(ScreenshotTest.kt:30)]]></stackTrace></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.io.IOException: The directory /storage/emulated/0/Pictures/screenshots does not exist and could not be created or is not writable.
	at androidx.test.runner.screenshot.BasicScreenCaptureProcessor.process(BasicScreenCaptureProcessor.java:72)
	at com.malinskiy.adam.junit4.android.screencapture.AdamScreenCaptureProcessor.process(AdamScreenCaptureProcessor.kt:47)
	at androidx.test.runner.screenshot.ScreenCapture.process(ScreenCapture.java:144)
	at androidx.test.runner.screenshot.ScreenCapture.process(ScreenCapture.java:126)
	at com.example.ScreenshotTest$testScreencapture$1$1.invoke(ScreenshotTest.kt:32)
	at com.example.ScreenshotTest$testScreencapture$1$1.invoke(ScreenshotTest.kt:31)
	at io.github.kakaocup.kakao.common.views.KDSLView.invoke(KDSLView.kt:12)
	at com.example.ScreenshotTest$testScreencapture$1.invoke(ScreenshotTest.kt:31)
	at com.example.ScreenshotTest$testScreencapture$1.invoke(ScreenshotTest.kt:30)
	at io.github.kakaocup.kakao.screen.Screen.invoke(Screen.kt:119)
	at com.example.ScreenshotTest.testScreencapture(ScreenshotTest.kt:30)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky8" time="0.822" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky8(MainActivityFlakyTest.kt:79)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky7" time="0.689" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky7(MainActivityFlakyTest.kt:73)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky4" time="1.73" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky4(MainActivityFlakyTest.kt:55)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky5" time="0.652" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky5(MainActivityFlakyTest.kt:61)]]></stackTrace></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky5(MainActivityFlakyTest.kt:61)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky2" time="0.637" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky2(MainActivityFlakyTest.kt:43)]]></stackTrace></flakyFailure><flakyFailure type="rerun exception"></flakyFailure></testcase><testcase name="testTextFlaky1" time="0.682" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky1(MainActivityFlakyTest.kt:37)]]></stackTrace></flakyFailure></testcase><testcase name="testThatWillNotSeeTheLightOfDay" time="0.0" classname="com.example.BeforeTestFailureTest"><failure><![CDATA[Test didn't complete. Either the test results are missing due to timeout or testing runtime crashed]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[Test didn't complete. Either the test results are missing due to timeout or testing runtime crashed]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[Test didn't complete. Either the test results are missing due to timeout or testing runtime crashed]]></stackTrace></rerunFailure></testcase><testcase name="test[6]" time="0.638" classname="com.example.ParameterizedTest"></testcase><testcase name="testAlwaysFailing" time="0.64" classname="com.example.TestWithWrongName"><failure><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.TestWithWrongName.testAlwaysFailing(TestWithWrongName.kt:17)]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.TestWithWrongName.testAlwaysFailing(TestWithWrongName.kt:17)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.TestWithWrongName.testAlwaysFailing(TestWithWrongName.kt:17)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.TestWithWrongName.testAlwaysFailing(TestWithWrongName.kt:17)]]></stackTrace></rerunFailure></testcase><testcase name="test[0]" time="0.641" classname="com.example.ParameterizedTest"></testcase><testcase name="test[1]" time="0.64" classname="com.example.ParameterizedTest"></testcase><testcase name="test[compute test for input 5: 5]" time="0.642" classname="com.example.NamedParameterizedTest"></testcase><testcase name="test[compute test for input 6: 8]" time="0.641" classname="com.example.NamedParameterizedTest"></testcase><testcase name="test[2]" time="0.633" classname="com.example.ParameterizedTest"></testcase><testcase name="test[3]" time="0.632" classname="com.example.ParameterizedTest"></testcase><testcase name="test[4]" time="0.633" classname="com.example.ParameterizedTest"></testcase><testcase name="test[5]" time="0.633" classname="com.example.ParameterizedTest"></testcase><testcase name="testAlwaysFailing" time="0.989" classname="com.example.AbstractFailingTest"><failure><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.AbstractFailingTest.testAlwaysFailing(AbstractFailingTest.kt:17)]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.AbstractFailingTest.testAlwaysFailing(AbstractFailingTest.kt:17)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.AbstractFailingTest.testAlwaysFailing(AbstractFailingTest.kt:17)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.AbstractFailingTest.testAlwaysFailing(AbstractFailingTest.kt:17)]]></stackTrace></rerunFailure></testcase><testcase name="test[compute test for input 0: 0]" time="0.653" classname="com.example.NamedParameterizedTest"></testcase><testcase name="test[compute test for input 1: 1]" time="0.647" classname="com.example.NamedParameterizedTest"></testcase><testcase name="test[compute test for input 2: 1]" time="0.648" classname="com.example.NamedParameterizedTest"></testcase><testcase name="test[compute test for input 3: 2]" time="0.648" classname="com.example.NamedParameterizedTest"></testcase><testcase name="test[compute test for input 4: 3]" time="0.647" classname="com.example.NamedParameterizedTest"></testcase><testcase name="testText6" time="1.503" classname="com.example.MainActivityTest"></testcase><testcase name="testText7" time="0.685" classname="com.example.MainActivityTest"></testcase><testcase name="testText8" time="0.328" classname="com.example.MainActivityTest"></testcase><testcase name="testText9" time="1.355" classname="com.example.MainActivityTest"></testcase><testcase name="testText" time="0.634" classname="com.example.MainActivityTest"></testcase><testcase name="testText1" time="1.696" classname="com.example.MainActivityTest"></testcase><testcase name="testText2" time="0.665" classname="com.example.MainActivityTest"></testcase><testcase name="testText3" time="0.665" classname="com.example.MainActivityTest"></testcase><testcase name="testText4" time="0.671" classname="com.example.MainActivityTest"></testcase><testcase name="testText5" time="0.676" classname="com.example.MainActivityTest"></testcase><testcase name="testWithAVeryLongNameThatExceeds255CharactersqwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM" time="2.554" classname="com.example.EdgeCaseTest"></testcase><testcase name="ignoreTest" time="0.678" classname="com.example.FailedAssumptionTest"><skipped></skipped></testcase><testcase name="failedAssumptionTest" time="0.685" classname="com.example.FailedAssumptionTest"><skipped><![CDATA[org.junit.AssumptionViolatedException: got: <false>, expected: is <true>
	at org.junit.Assume.assumeThat(Assume.java:106)
	at org.junit.Assume.assumeTrue(Assume.java:50)
	at com.example.FailedAssumptionTest.failedAssumptionTest(FailedAssumptionTest.kt:19)
	at java.lang.reflect.Method.invoke(Native Method)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at androidx.test.ext.junit.runners.AndroidJUnit4.run(AndroidJUnit4.java:162)
	at io.qameta.allure.android.runners.AllureAndroidJUnit4.run(AllureAndroidJUnitRunners.kt:29)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
	at androidx.test.internal.runner.TestExecutor.execute(TestExecutor.java:67)
	at androidx.test.internal.runner.TestExecutor.execute(TestExecutor.java:58)
	at androidx.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:446)
	at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:2361)]]></skipped></testcase><testcase name="testVmState" time="0.661" classname="com.example.GrpcActivityTest"></testcase><testcase name="testExternalFilePresent" time="1.102" classname="com.example.FilePushTest"><failure><![CDATA[java.io.FileNotFoundException: /data/local/tmp/external-file: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:574)
	at java.io.FileInputStream.<init>(FileInputStream.java:160)
	at kotlin.io.FilesKt__FileReadWriteKt.readText(FileReadWrite.kt:125)
	at kotlin.io.FilesKt__FileReadWriteKt.readText$default(FileReadWrite.kt:125)
	at com.example.FilePushTest.testExternalFilePresent(FilePushTest.kt:16)
	... 32 trimmed
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:274)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7758)
	at libcore.io.IoBridge.open(IoBridge.java:560)
	... 37 more]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.io.FileNotFoundException: /data/local/tmp/external-file: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:496)
	at java.io.FileInputStream.<init>(FileInputStream.java:159)
	at kotlin.io.FilesKt__FileReadWriteKt.readText(FileReadWrite.kt:125)
	at kotlin.io.FilesKt__FileReadWriteKt.readText$default(FileReadWrite.kt:125)
	at com.example.FilePushTest.testExternalFilePresent(FilePushTest.kt:16)
	... 32 trimmed
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7255)
	at libcore.io.IoBridge.open(IoBridge.java:482)
	... 37 more]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.io.FileNotFoundException: /data/local/tmp/external-file: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:492)
	at java.io.FileInputStream.<init>(FileInputStream.java:160)
	at kotlin.io.FilesKt__FileReadWriteKt.readText(FileReadWrite.kt:125)
	at kotlin.io.FilesKt__FileReadWriteKt.readText$default(FileReadWrite.kt:125)
	at com.example.FilePushTest.testExternalFilePresent(FilePushTest.kt:16)
	... 32 trimmed
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:166)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:254)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:166)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7542)
	at libcore.io.IoBridge.open(IoBridge.java:478)
	... 37 more]]></stackTrace></rerunFailure></testcase><testcase name="testExternalFolderPresent" time="0.691" classname="com.example.FilePushTest"><failure><![CDATA[java.io.FileNotFoundException: /data/local/tmp/external-folder/external-file-2: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:574)
	at java.io.FileInputStream.<init>(FileInputStream.java:160)
	at kotlin.io.FilesKt__FileReadWriteKt.readText(FileReadWrite.kt:125)
	at kotlin.io.FilesKt__FileReadWriteKt.readText$default(FileReadWrite.kt:125)
	at com.example.FilePushTest.testExternalFolderPresent(FilePushTest.kt:22)
	... 32 trimmed
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:274)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:563)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7758)
	at libcore.io.IoBridge.open(IoBridge.java:560)
	... 37 more]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.io.FileNotFoundException: /data/local/tmp/external-folder/external-file-2 (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:231)
	at java.io.FileInputStream.<init>(FileInputStream.java:165)
	at kotlin.io.FilesKt__FileReadWriteKt.readText(FileReadWrite.kt:125)
	at kotlin.io.FilesKt__FileReadWriteKt.readText$default(FileReadWrite.kt:125)
	at com.example.FilePushTest.testExternalFolderPresent(FilePushTest.kt:22)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.io.FileNotFoundException: /data/local/tmp/external-folder/external-file-2: open failed: ENOENT (No such file or directory)
	at libcore.io.IoBridge.open(IoBridge.java:496)
	at java.io.FileInputStream.<init>(FileInputStream.java:159)
	at kotlin.io.FilesKt__FileReadWriteKt.readText(FileReadWrite.kt:125)
	at kotlin.io.FilesKt__FileReadWriteKt.readText$default(FileReadWrite.kt:125)
	at com.example.FilePushTest.testExternalFolderPresent(FilePushTest.kt:22)
	... 32 trimmed
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
	at libcore.io.Linux.open(Native Method)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
	at libcore.io.BlockGuardOs.open(BlockGuardOs.java:252)
	at libcore.io.ForwardingOs.open(ForwardingOs.java:167)
	at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7255)
	at libcore.io.IoBridge.open(IoBridge.java:482)
	... 37 more]]></stackTrace></rerunFailure></testcase><testcase name="testTextFlaky" time="1.115" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky(MainActivityFlakyTest.kt:31)]]></stackTrace></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky(MainActivityFlakyTest.kt:31)]]></stackTrace></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky(MainActivityFlakyTest.kt:31)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky3" time="0.652" classname="com.example.MainActivityFlakyTest"><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky3(MainActivityFlakyTest.kt:49)]]></stackTrace></flakyFailure><flakyFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at com.example.MainActivityFlakyTest.testTextFlaky3(MainActivityFlakyTest.kt:49)]]></stackTrace></flakyFailure></testcase><testcase name="testTextFlaky6" time="0.652" classname="com.example.MainActivityFlakyTest"></testcase><testcase name="testText" time="0.653" classname="com.example.MainActivityAllureTest"></testcase><testcase name="testFailure" time="0.661" classname="com.example.MainActivityAllureTest"><failure><![CDATA[androidx.test.espresso.base.AssertionErrorHandler$AssertionFailedWithCauseError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=1584, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=0 initialSelEnd=0 initialCapsMode=0x0 hintText=null label=null packageName=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=false, has-links=false}
	at dalvik.system.VMStack.getThreadStackTrace(Native Method)
	at java.lang.Thread.getStackTrace(Thread.java:1720)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:3)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:1)
	at androidx.test.espresso.base.DefaultFailureHandler$TypedFailureHandler.handle(DefaultFailureHandler.java:4)
	at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:5)
	at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:5)
	at androidx.test.espresso.ViewInteraction.check(ViewInteraction.java:12)
	at io.github.kakaocup.kakao.delegate.ViewInteractionDelegate.check(ViewInteractionDelegate.kt:26)
	at io.github.kakaocup.kakao.text.TextViewAssertions$DefaultImpls.hasText(TextViewAssertions.kt:58)
	at io.github.kakaocup.kakao.edit.KEditText.hasText(KEditText.kt:18)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:59)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:58)
	at io.qameta.allure.kotlin.Allure.step(Allure.kt:60)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:58)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:57)
	at io.github.kakaocup.kakao.common.views.KDSLView.invoke(KDSLView.kt:12)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:57)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:56)
	at io.github.kakaocup.kakao.screen.Screen.invoke(Screen.kt:119)
	at com.example.MainActivityAllureTest.testFailure(MainActivityAllureTest.kt:56)
	... 37 trimmed
Caused by: junit.framework.AssertionFailedError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=1584, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=0 initialSelEnd=0 initialCapsMode=0x0 hintText=null label=null packageName=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=false, has-links=false}
	at androidx.test.espresso.matcher.ViewMatchers.assertThat(ViewMatchers.java:16)
	at androidx.test.espresso.assertion.ViewAssertions$MatchesViewAssertion.check(ViewAssertions.java:7)
	at androidx.test.espresso.ViewInteraction$SingleExecutionViewAssertion.check(ViewInteraction.java:2)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:14)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:1)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[androidx.test.espresso.base.AssertionErrorHandler$AssertionFailedWithCauseError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=2127, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=0 initialSelEnd=0 initialCapsMode=0x0 hintText=null label=null packageName=null autofillId=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=false, has-links=false}
	at dalvik.system.VMStack.getThreadStackTrace(Native Method)
	at java.lang.Thread.getStackTrace(Thread.java:1841)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:3)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:1)
	at androidx.test.espresso.base.DefaultFailureHandler$TypedFailureHandler.handle(DefaultFailureHandler.java:4)
	at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:5)
	at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:5)
	at androidx.test.espresso.ViewInteraction.check(ViewInteraction.java:12)
	at io.github.kakaocup.kakao.delegate.ViewInteractionDelegate.check(ViewInteractionDelegate.kt:26)
	at io.github.kakaocup.kakao.text.TextViewAssertions$DefaultImpls.hasText(TextViewAssertions.kt:58)
	at io.github.kakaocup.kakao.edit.KEditText.hasText(KEditText.kt:18)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:59)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:58)
	at io.qameta.allure.kotlin.Allure.step(Allure.kt:60)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:58)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:57)
	at io.github.kakaocup.kakao.common.views.KDSLView.invoke(KDSLView.kt:12)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:57)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:56)
	at io.github.kakaocup.kakao.screen.Screen.invoke(Screen.kt:119)
	at com.example.MainActivityAllureTest.testFailure(MainActivityAllureTest.kt:56)
	... 37 trimmed
Caused by: junit.framework.AssertionFailedError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=2127, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=0 initialSelEnd=0 initialCapsMode=0x0 hintText=null label=null packageName=null autofillId=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=false, has-links=false}
	at androidx.test.espresso.matcher.ViewMatchers.assertThat(ViewMatchers.java:16)
	at androidx.test.espresso.assertion.ViewAssertions$MatchesViewAssertion.check(ViewAssertions.java:7)
	at androidx.test.espresso.ViewInteraction$SingleExecutionViewAssertion.check(ViewInteraction.java:2)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:14)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:1)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7872)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[androidx.test.espresso.base.AssertionErrorHandler$AssertionFailedWithCauseError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=1584, has-focus=true, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=true, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=4 initialSelEnd=4 initialCapsMode=0x0 hintText=null label=null packageName=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=true, has-links=false}
	at dalvik.system.VMStack.getThreadStackTrace(Native Method)
	at java.lang.Thread.getStackTrace(Thread.java:1538)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:3)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:1)
	at androidx.test.espresso.base.DefaultFailureHandler$TypedFailureHandler.handle(DefaultFailureHandler.java:4)
	at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:5)
	at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:5)
	at androidx.test.espresso.ViewInteraction.check(ViewInteraction.java:12)
	at io.github.kakaocup.kakao.delegate.ViewInteractionDelegate.check(ViewInteractionDelegate.kt:26)
	at io.github.kakaocup.kakao.text.TextViewAssertions$DefaultImpls.hasText(TextViewAssertions.kt:58)
	at io.github.kakaocup.kakao.edit.KEditText.hasText(KEditText.kt:18)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:59)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:58)
	at io.qameta.allure.kotlin.Allure.step(Allure.kt:60)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:58)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:57)
	at io.github.kakaocup.kakao.common.views.KDSLView.invoke(KDSLView.kt:12)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:57)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:56)
	at io.github.kakaocup.kakao.screen.Screen.invoke(Screen.kt:119)
	at com.example.MainActivityAllureTest.testFailure(MainActivityAllureTest.kt:56)
	... 37 trimmed
Caused by: junit.framework.AssertionFailedError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=1584, has-focus=true, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=true, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=4 initialSelEnd=4 initialCapsMode=0x0 hintText=null label=null packageName=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=true, has-links=false}
	at androidx.test.espresso.matcher.ViewMatchers.assertThat(ViewMatchers.java:16)
	at androidx.test.espresso.assertion.ViewAssertions$MatchesViewAssertion.check(ViewAssertions.java:7)
	at androidx.test.espresso.ViewInteraction$SingleExecutionViewAssertion.check(ViewInteraction.java:2)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:14)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:1)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6669)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[androidx.test.espresso.base.AssertionErrorHandler$AssertionFailedWithCauseError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=1584, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=0 initialSelEnd=0 initialCapsMode=0x0 hintText=null label=null packageName=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=false, has-links=false}
	at dalvik.system.VMStack.getThreadStackTrace(Native Method)
	at java.lang.Thread.getStackTrace(Thread.java:1720)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:3)
	at androidx.test.espresso.base.AssertionErrorHandler.handleSafely(AssertionErrorHandler.java:1)
	at androidx.test.espresso.base.DefaultFailureHandler$TypedFailureHandler.handle(DefaultFailureHandler.java:4)
	at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:5)
	at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:5)
	at androidx.test.espresso.ViewInteraction.check(ViewInteraction.java:12)
	at io.github.kakaocup.kakao.delegate.ViewInteractionDelegate.check(ViewInteractionDelegate.kt:26)
	at io.github.kakaocup.kakao.text.TextViewAssertions$DefaultImpls.hasText(TextViewAssertions.kt:58)
	at io.github.kakaocup.kakao.edit.KEditText.hasText(KEditText.kt:18)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:59)
	at com.example.MainActivityAllureTest$testFailure$1$1$1.invoke(MainActivityAllureTest.kt:58)
	at io.qameta.allure.kotlin.Allure.step(Allure.kt:60)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:58)
	at com.example.MainActivityAllureTest$testFailure$1$1.invoke(MainActivityAllureTest.kt:57)
	at io.github.kakaocup.kakao.common.views.KDSLView.invoke(KDSLView.kt:12)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:57)
	at com.example.MainActivityAllureTest$testFailure$1.invoke(MainActivityAllureTest.kt:56)
	at io.github.kakaocup.kakao.screen.Screen.invoke(Screen.kt:119)
	at com.example.MainActivityAllureTest.testFailure(MainActivityAllureTest.kt:56)
	... 37 trimmed
Caused by: junit.framework.AssertionFailedError: 'an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"' doesn't match the selected view.
Expected: an instance of android.widget.TextView and view.getText() with or without transformation to match: is "Cafebabe"
     Got: view.getText() was "Test"
View Details: EditText{id=2131230861, res-name=edittext, visibility=VISIBLE, width=1080, height=1584, has-focus=false, has-focusable=true, has-window-focus=true, is-clickable=true, is-enabled=true, is-focused=false, is-focusable=true, is-layout-requested=false, is-selected=false, layout-params=androidx.constraintlayout.widget.ConstraintLayout$LayoutParams@YYYYYY, tag=null, root-is-layout-requested=false, has-input-connection=true, editor-info=[inputType=0x20001 imeOptions=0x40000006 privateImeOptions=null actionLabel=null actionId=0 initialSelStart=0 initialSelEnd=0 initialCapsMode=0x0 hintText=null label=null packageName=null fieldId=0 fieldName=null extras=null hintLocales=null contentMimeTypes=null ], x=0.0, y=0.0, text=Test, input-type=131073, ime-target=false, has-links=false}
	at androidx.test.espresso.matcher.ViewMatchers.assertThat(ViewMatchers.java:16)
	at androidx.test.espresso.assertion.ViewAssertions$MatchesViewAssertion.check(ViewAssertions.java:7)
	at androidx.test.espresso.ViewInteraction$SingleExecutionViewAssertion.check(ViewInteraction.java:2)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:14)
	at androidx.test.espresso.ViewInteraction$2.call(ViewInteraction.java:1)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at android.os.Handler.handleCallback(Handler.java:883)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)]]></stackTrace></rerunFailure></testcase><testcase name="testVmState" time="1.247" classname="com.example.ConsoleActivityTest"><failure><![CDATA[java.lang.AssertionError: Assertion failed
	at com.example.ConsoleActivityTest$testVmState$1.invokeSuspend(ConsoleActivityTest.kt:32)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:202)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:279)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at com.example.ConsoleActivityTest.testVmState(ConsoleActivityTest.kt:29)]]></failure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError: Assertion failed
	at com.example.ConsoleActivityTest$testVmState$1.invokeSuspend(ConsoleActivityTest.kt:32)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:202)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:279)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at com.example.ConsoleActivityTest.testVmState(ConsoleActivityTest.kt:29)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError: Assertion failed
	at com.example.ConsoleActivityTest$testVmState$1.invokeSuspend(ConsoleActivityTest.kt:32)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:202)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:279)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at com.example.ConsoleActivityTest.testVmState(ConsoleActivityTest.kt:29)]]></stackTrace></rerunFailure><rerunFailure type="rerun exception"><stackTrace><![CDATA[java.lang.AssertionError: Assertion failed
	at com.example.ConsoleActivityTest$testVmState$1.invokeSuspend(ConsoleActivityTest.kt:32)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.UndispatchedCoroutine.afterResume(CoroutineContext.kt:202)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:279)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:1)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:1)
	at com.example.ConsoleActivityTest.testVmState(ConsoleActivityTest.kt:29)]]></stackTrace></rerunFailure></testcase><testcase name="null" time="0.65" classname="com.example.ClassIgnoredTest"><skipped></skipped></testcase><testcase name="testUnsafeAccess" time="1.208" classname="com.example.AdbActivityTest"></testcase><testcase name="testVmState" time="1.198" classname="com.example.AdbActivityTest"></testcase><testcase name="testTextSlow1" time="6.094" classname="com.example.MainActivitySlowTest"></testcase><testcase name="testTextSlow2" time="5.24" classname="com.example.MainActivitySlowTest"></testcase><testcase name="testTextSlow3" time="5.151" classname="com.example.MainActivitySlowTest"></testcase><testcase name="testTextSlow4" time="5.284" classname="com.example.MainActivitySlowTest"></testcase><testcase name="testTextSlow" time="5.296" classname="com.example.MainActivitySlowTest"></testcase></testsuite>